src/isoStringScalar/isoStringScalar.ts

```diff
@@ -0,0 +1,93 @@
+import { GraphQLScalarType, Kind } from 'graphql';
+import { DateTime } from 'luxon';
+import {
+  InternalServerError,
+  UserInputError,
+} from '../errorHandler/errorHandler';
+
+export const isoStringScalar = new GraphQLScalarType({
+  name: 'ISOString',
+  description: 'ISOString custom scalar type',
+
+  /**
+   * Converts server-side TS Date representation to a JSON-compatible, ISO-8601-compliant
+   * UTC Datetime String for Apollo Server to include in an operation response.
+   *
+   * @param value - TS Date Object as generated by Data Store Client & validated by mysqlDateConvert.
+   * @returns ISO-8601-compliant, UTC-based Datetime String.
+   */
+  serialize(value: Date | null): string | null {
+    if (value === null) {
+      return null;
+    }
+
+    if (!(value instanceof Date)) {
+      throw new InternalServerError(
+        'GraphQL ISOString Scalar serializer expected a `Date` object or null'
+      );
+    }
+
+    // isNaN here checks for 0000-00-00 style dates, which are invalid
+    if (isNaN(value.valueOf())) {
+      throw new InternalServerError(
+        'Invalid Data Store Response: invalid Date object'
+      );
+    }
+
+    // toISOString() is always zero UTC offset per
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+    const resp: string = value.toISOString();
+    return resp;
+  },
+
+  /**
+   * Converts the scalar's JSON value to its back-end representation
+   * before it's added to a resolver's args.
+   *
+   * @param value - ISO-8601-compliant Datetime String.
+   * @returns TS Data Object for interacting with Data Store via Client.
+   */
+  parseValue(value: string | null): Date | null {
+    if (value === null || value === '') {
+      return null;
+    }
+
+    if (!(typeof value === 'string')) {
+      throw new UserInputError(
+        'Invalid User Input: ISOString Scalar parse expected a value of type string or null'
+      );
+    }
+
+    const isoDateTime = DateTime.fromISO(value);
+    if (!isoDateTime.isValid) {
+      // if not ISO, try SQL Date Form, forced to UTC
+      const sqlDateTime = DateTime.fromSQL(value, { zone: 'UTC' });
```

Comment:
Our db timezone is actually in central time, and the strings don't have timezone on them (yes... I know...) -- recommend exposing this in the API so we can handle any db timezone

Reply:
@kschelonka that does not align with the scope of this. This UTC is forcing SQL-style dates given to us _by the client_ - it has nothing to do with the database.

Reply:
thank you for the reminder, I think the .fromSQL method confused me which function I was reading.

Reply:
discussed in slack, removed this